### DISCUSSION TITLE
Cannot move Kanboard inside a subfolder

### DISCUSSION THREAD

**Discussion by aario** [kanboard/kanboard#5192] [Installation & Configuration]
### Actual behaviour

If I move kanboard inside a subfolder on my webserver server, all assets still have addresses starting with '/'. For example:
```
/assets/css/vendor.min.css?1574224047
```
So they point to root folder of the server. I could not find any configuration in config.php nor any information on documentation how to disable this leading `/` in asset paths.

### Expected behaviour

Asset paths should not be absolute.

### Steps to reproduce

Install kanboard inside a subfolder so it can be accessed like:
```
https://mycompany.com/kanboard/
```
### Configuration

-     Application version: v1.2.13
-     PHP version: 7.3.11
-     PHP SAPI: fpm-fcgi
-     HTTP Client: cURL
-     OS version: Linux 4.19.0-8-amd64
-     Database driver: postgres
-     Database version: 12.2
-     Browser: Mozilla/5.0 (X11; Linux x86_64; rv:68.0) Gecko/20100101 Firefox/68.0

**Comment by fguillot** [comment]
Try to define the application URL in the application settings.

**Comment by Floyer007** [comment]
I've a simmilar issue and it seems, that the "Application URL" has no effect.

**Setup:**
Kanboard runs from docker-container on localhost:8080.
Nginx should proxy from https://mydomain.tld/kanboard to localhost:8080.
Application-URL is set to: https://mydomain.tld/kanboard/

https://mydomain.tld/kanboard itself works, unless I click onto a link (e.g. settings).
The url doesn't direct to https://mydomain.tld/kanboard/settings.
Instead it changes to https://mydomain.tld/settings

**Comment by stertingen** [comment]
I think this function is to blame here:

https://github.com/kanboard/kanboard/blob/master/app/Helper/UrlHelper.php#L171

`PHP_SELF` is the wrong variable to query here. To determine the application root URL, the application should consult the configuration variable instead.

Also, the `KANBOARD_URL` define is barely documented, how am I supposed to use it?

**Comment by stertingen** [comment]
`define('KANBOARD_URL', 'https://mydomain.tld/kanboard/');` into the config did the trick.
